src/main/java/racingcar/Application.java

```diff
@@ -0,0 +1,23 @@
+package racingcar;
+
+import racingcar.controller.RacingGame;
+import racingcar.domain.RoundResult;
+import racingcar.domain.Winners;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+public class Application {
+    public static void main(String[] args) {
+        RacingGame racingGame = new RacingGame(InputView.inputNames(), InputView.inputRound());
+
+        OutputView.printResultIntro();
+        while (!racingGame.isFinished()) {
+            racingGame.race();
+            RoundResult roundResult = racingGame.getRoundResult();
+            OutputView.printRoundResult(roundResult.getPositions());
```

Comment:
racingGame이 모든 걸 가지고 있네요. 
그렇다면 roundResult를 가져오지 않고 racingGame에게 물어보면 되지 않을까요?

```suggestion
            OutputView.printRoundResult(racingGame.getPositions());
```

Reply:
반영했습니다~